README.md

```diff
@@ -1,4 +1,16 @@
 # [Resource gallery🛸](https://resourcegallery.live/)
+
+<div align="center">
+<img src="https://forthebadge.com/images/badges/built-with-love.svg" />
+<img src="https://forthebadge.com/images/badges/uses-brains.svg" />
```

Comment:
change uses-astro
&
remove responsibility badge

Reply:
This cant be customized..other standard templates are there..such as powered by coffee,powered by pull requests etc.Do you want me to remove it?

Reply:
which generator you used?

Reply:
For the badge

Reply:
You can generate custom one too from there. try to generate and add here